hamocc/mo_shelfsea_restime.F90

```diff
@@ -0,0 +1,73 @@
+! Copyright (C) 2021-2022  J. Schwinger
```

Comment:
Update this?

Reply:
You mean to 2021-2024? (- or do you mean your name?)

Reply:
Both

Reply:
I did now, but maybe we can have a short conversation on this at some point - thus far, I never updated those entries in any of the code, where I was working on in iHAMOCC (at least not in what was present) - I see that in BLOM, this is somewhat handled differently. I guess, I to this date see the PRs as main documentation of this (and not being sure, how the copyright holding note should be handled as long as it is one codebase). Let's have a chat together, when all being back.

Reply:
I also always forget to update this for changes made to existing files. For new files, I think I have usually updated it. Here I just found it weird to have my name in a routine that was written by someone else... I'm happy to have a chat